data/nlu/nlu.md

```diff
@@ -225,6 +223,61 @@
 - yup
 - yyeeeh
 - considering
+- More a less
+- cool beans
+
+## intent:react_positive
+- good
+- nice
```

Comment:
I think i'll revert these. but just wanted to point out that these can sometimes be a react positive, like
"rasa is open source!"
"nice"